src/lib/y2storage/callbacks/libstorage_callback.rb

```diff
@@ -90,6 +96,49 @@ def abort_button_label
       def default_answer_to_error
         true
       end
+
+      # Human-readable description of the problem reported by libstorage-ng,
+      # hopefully with some hint on how to resolve it or continue.
+      #
+      # A generic message is returned if no concrete problem can be identified.
+      #
+      # @see #error
+      #
+      # @param what [String] details coming from libstorage-ng
+      # @return [String]
+      def error_description(what)
+        if what.match?(/WARNING: PV .* was already found on .*/i)
+          duplicated_pv_description
+        else
+          _("Unexpected situation found in the system.")
+        end
+      end
+
+      # Human-readable description to use if the LVM tools report that the same
+      # PV is found more than once.
+      #
+      # @see #error_description
+      #
+      # @return [String]
+      def duplicated_pv_description
+        result = _(
+          "The same LVM physical volume was found in several devices.\n" \
+          "Maybe there are multipath devices in the system but multipath support\n" \
+          "was not enabled.\n\n"
+        )
+        if Yast::Mode.auto
+          result << _(
+            "Use 'start_multipath' in the AutoYaST profile to enable multipath."
+          )
+        else
+          result << _(
```

Comment:
NP: I think we can detect if the user already has used `LIBSTORAGE_MULTIPATH_AUTOSTART` and, in that case, change the message a little. Otherwise, the user could think that he is not doing it correctly.